sapp/sarif.py

```diff
@@ -48,7 +48,7 @@
 
 class SARIF:
     version: str = "2.1.0"
-    schema: str = "https://raw.githubusercontent.com/oasis-tcs/sarif-spec/master/Schemata/sarif-schema-2.1.0.json"  # noqa
```

Comment:
the old schema is actually had a broken regex and it was fixed in the codeQL schema see this commit https://github.com/github/codeql-action/commit/9824588133570daa5317a5d985c3c181c80c4354